screens/Registration/InterestsScreen/styles.ts

```diff
@@ -116,9 +133,9 @@ const styles = StyleSheet.create({
 
   title: {
     fontSize: 24,
-    fontWeight: 'bold',
+    fontWeight: "bold",
```

Comment:
Instead of using fontWeight, you can just use add `globalStyles.boldText` in the styles of your component, try to use our custom fonts for all text.

Reply:
Nice catch! Thanks